src/ts/components/playbackspeedselectbox.ts

```diff
@@ -38,5 +33,33 @@ export class PlaybackSpeedSelectBox extends SelectBox {
 
     // when the player hits onReady again, adjust the playback speed selection with fallback to default 1
     player.addEventHandler(player.EVENT.ON_READY, setDefaultValue);
+
+    if (player.EVENT.ON_PLAYBACK_SPEED_CHANGED) {
+      // Since player 7.7.0
+      player.addEventHandler(player.EVENT.ON_PLAYBACK_SPEED_CHANGED, () => {
+        const speed = String(player.getPlaybackSpeed());
+
+        if (!this.selectItem(speed)) {
+          // a playback speed was set which is not in the list, add it to the list to show it to the user
+          this.clearItems();
```

Comment:
Why do we clear everything here and then add the default items + the new one? Wouldn't it be enough to just add the new Item.
We should also add the playback speed to the list when we set the initial value in line 26:
```ts
       let playbackSpeed = String(player.getPlaybackSpeed());
       if (!this.selectItem(playbackSpeed)) {
         playbackSpeed = '1';
```
Furthermore i think the we should do a sorted insert to improve the usability

Reply:
If multiple "non-standard" values are selected, each time the item would be added and the list could grow. Therefore I decided to clear the old ones, but open for discussion.

Good catch with the initial value in L26, will update.

Sorted insert would have been nice... 😉

Reply:
Updated to have sorted insert as well.